ansible/roles/bootstrap/tasks/main.yml

```diff
@@ -1,5 +1,27 @@
 #Deploys Bootstrap Node
 
+- name: Install base packages
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - "docker.io"
+    - "htop"
+    - "vim"
+    - "ethtool"
+    - "tmux"
```

Comment:
i think most developers would feel `tmux` is a requirement :)

but is it needed for the deployment?